tutorials/examples/train_line.py

```diff
@@ -336,11 +236,11 @@ def train(
         loss.backward()
 
         # Gradient Clipping.
-        for p in gflownet.parameters():
-            if p.ndim > 0 and p.grad is not None:  # We do not clip logZ grad.
-                p.grad.data.clamp_(
-                    -gradient_clip_value, gradient_clip_value
-                ).nan_to_num_(0.0)
+        # for p in gflownet.parameters():
+        #     if p.ndim > 0 and p.grad is not None:  # We do not clip logZ grad.
+        #         p.grad.data.clamp_(
+        #             -gradient_clip_value, gradient_clip_value
+        #         ).nan_to_num_(0.0)
```

Comment:
I would suggest deleting this code or adding a comment explaining why it's not included in the example.

Reply:
Oups -- this is a mistake - it shouldn't be commented out - good catch :)